plugins/catalog-backend-module-openapi-spec/README.md

```diff
@@ -0,0 +1,30 @@
+# catalog-backend-module-openapi-spec
+
+## Summary
+
+This module installs an entity provider that exports a single entity, your Backstage instance documentation, which merges as many backend plugins as you have defined in the config value `openapi.plugins`.
+
+## Notes
+
+- This only works with the new backend system.
+- This requires populating a new config value `openapi.plugins` with an array of plugin IDs. The entity processor expo
+
+## Installation
+
+To your new backend file, add
+
+```ts title="packages/backend/src/index.ts"
+backend.add(import('@backstage/plugin-catalog-backend-module-openapi-spec'));
+```
+
+Add a list of plugins to your config like,
+
+```yaml title="app-config.yaml"
+openapi:
+  plugins:
+    - catalog
+    - todo
+    - search
```

Comment:
does that mean that there'd have to be a `@backstage/plugin-todo-backend-module-openapi-spec` and a `@backstage/plugin-search-backend-module-openapi-spec` too?

as in, will any backend plugin need also an openapi spec module too, and it's just that catalog was a  place to start?

Reply:
The naming here is a bit unfortunate as we already have a `@backstage/plugin-catalog-backend-module-openapi` that pulls in OpenAPI specs from `$ref`s. This is a module to add all of the internal registered specs into the catalog. You wouldn't need a `@backstage/plugin-todo-backend-module-openapi-spec` as long as you list `todo` or `search` in the config under `openapi.plugins`.